src/System.Management.Automation/engine/lang/interface/PSParseError.cs

```diff
@@ -1,5 +1,5 @@
 /********************************************************************++
-    Copyright (C) Microsoft Corporation, 2003
+    Copyright (c) Microsoft Corporation, 2003
```

Comment:
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."